src/cli.rs

```diff
@@ -195,6 +199,10 @@ pub enum Commands {
         /// Number of CPU threads
         #[arg(long, value_parser = valid_cpus, default_value_t = 1)]
         threads: usize,
+
+        /// Number of reads before stopping
+        #[arg(long)]
+        max_reads: Option<usize>,
```

Comment:
I would suggest we just have this as an option in `build` as that's for any 'advanced' use, whereas providing files with align/map is for quick use with the defaults. So would remove these two uses in the CLI, and this will also simplify some options below